src/unix/udp.c

```diff
@@ -263,11 +263,29 @@ static void uv__udp_sendmsg(uv_udp_t* handle) {
  * are different from the BSDs: it _shares_ the port rather than steal it
  * from the current listener.  While useful, it's not something we can emulate
  * on other platforms so we don't enable it.
+ *
+ * zOS does not support getsockname with SO_REUSEPORT option when using 
+ * AF_UNIX  
  */
 static int uv__set_reuse(int fd) {
   int yes;
 
-#if defined(SO_REUSEPORT) && !defined(__linux__)
+#if defined(__MVS__) && defined(SO_REUSEPORT)                  
```

Comment:
nit: maybe reorder this so SO_REUSEPORT is first, as it is in the other conditional? Just for consistency, the compiler won't care.